content/docs/getting-started/getting-started-for-admins/index.md

```diff
@@ -25,66 +25,18 @@ In order to complete this guide, you will need the following things:
 2. A GitHub Cloud or GitHub Enterprise Cloud account.
 3. A test service on GitHub which you can make changes to.
 
-## Connect Roadie to GitHub
+## Install the Roadie GitHub App
 
-Roadie needs 2 connections to GitHub in order to function.
+Roadie needs the "Roadie Backstage" GitHub app to be installed in your GitHub organization so that it can index your repositories.
 
-1.  A Personal Access Token to access the YAML metadata files that Backstage needs to operate.
-2.  An OAuth client ID and secret so that your user can make requests from the browser to the GitHub API.
+You can find a link to install the GitHub app in the Backstage application:
 
-Use the steps below to create a client ID and secret and Personal Access Token (PAT) for Roadie.
+1. Click the Adminisatration settings link in the bottom left
+2. Click the Integrations tab along the top of the Administration settings page
+3. Click on the link to install the GitHub app.
 
 If you wish to tightly restrict the access that Roadie has, perhaps for evaluation purposes, please [follow this guide](/docs/github-token/).
 
-### Step 1. Create a Personal Access Token (PAT)
-
-Visit the [developer settings of your GitHub account](https://github.com/settings/tokens) and create a PAT which has the scopes `repo`, `user`, `read:org` and `read:discussion`.
-
-The official GitHub docs for creating a Personal Access Token are [here](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
-
-⚠️ &nbsp;The token's owner must be a member of the organization, rather than an outside collaborator or other relation.
-
-Want to use a token with more restrictive scopes? Use [this guide](/docs/integrations/github-token/).
-
-### Step 2. Create OAuth Credentials
```

Comment:
GitHub apps can only have a [max of 10 callback URLs](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app).

Given this, it may be premature to remove this section. We can only do it ourselves for 5 customers. We are already almost at the limit.

Perhaps keep this as the standard workflow until we have removed the need to set callback URLs for each tenant?

Reply:
done